config/config.go

```diff
@@ -71,6 +71,8 @@ type Configuration struct {
 	Debug Debug `mapstructure:"debug"`
 	// RequestValidation specifies the request validation options.
 	RequestValidation RequestValidation `mapstructure:"request_validation"`
+	// If req.Source.TID should be filled with a randomly generated UUID or not, if it is empty
+	AutoSourceTIDFill bool `mapstructure:"auto_source_tid_fill"`
```

Comment:
I was torn between `AutoSourceTIDFill` and `RequireSourceTID` as the name for this config. Please lemme know if `RequireSourceTID` sounds better or if you have any other suggestions for this one.